### DISCUSSION TITLE
奇偶页的内容存在一个字符的整体偏移

### DISCUSSION THREAD

**Discussion by WXiaoyao** [note286/xduts#47] [Q&A] [answered]
在使用enumerate环境的时候偶然发现了这个问题。下图是enumerate环境的一个item项：
![image](https://user-images.githubusercontent.com/28039355/219378516-e38c044a-3ee5-4297-8a92-c084db6989ba.png)
但是按理说跨页之后的第一个“密”字应该是和上一页的“U”和“钥”字对齐的，但实际却是和item项的标号对齐：
![image](https://user-images.githubusercontent.com/28039355/219378154-f33af70d-9cf7-4a1c-9c6e-5f57faa09344.png)
之后往上翻的时候发现，发现这是一个整体现象，即奇偶页面的内容是存在整体的偏移的：
![image](https://user-images.githubusercontent.com/28039355/219380709-b34ffd34-c6bb-4df1-acec-0280daafc528.png)
这个算是bug吗

**Comment by note286** [comment, marked as answer]
见#44。

**Reply by WXiaoyao** [reply]
可以麻烦给一下奇偶页边距不同的原文出处吗
我仔仔细细读了好几遍也没找到，就看到下面这个，感觉也没说
![image](https://user-images.githubusercontent.com/28039355/219386573-56a45320-c123-4c1b-9813-7ca6daf4a7a8.png)
真的挺好奇的。。

**Reply by note286** [reply]
学位论文是要装订的，你打开一本书，去观察一下相邻左右两页的页边距就知道为啥了。

**Reply by WXiaoyao** [reply]
好的好的明白了，感谢答疑~

**Reply by note286** [reply]
平时在屏幕上查看，可以开启对称页边距，PDF文档里有提供接口。

**Reply by WXiaoyao** [reply]
嗯嗯好的

**Reply by sikouhjw** [reply]
> 可以麻烦给一下奇偶页边距不同的原文出处吗 我仔仔细细读了好几遍也没找到，就看到下面这个，感觉也没说 ![image](https://user-images.githubusercontent.com/28039355/219386573-56a45320-c123-4c1b-9813-7ca6daf4a7a8.png) 真的挺好奇的。。

『装订线为 0.5 厘米』

**Reply by WXiaoyao** [reply]
好的谢谢~我昨晚去翻了下书确实想明白了，因为为了方便阅读，正文与两边的页边距应该是不一样的（应该更往外偏一些），又因为奇偶页是正反页的关系，它们的的左右边距是反过来的，所以在一个pdf里阅读的时候奇偶页的内容就会有偏移。这个问题我以前从来没想过，也算是涨知识了哈哈

**Reply by note286** [reply]
PDF阅读器，都有书籍视图模式，可以双页。

**Reply by WXiaoyao** [reply]
好的谢谢
